# Small Assignment 2 Solutions

The following model is a simplified version of the multiple regression model used by Biddle and
Hammermesh (1990) to study the tradeoff between time spent sleeping and working and to
look at other factors affecting sleep

$\textrm{sleep}=\beta_0+\beta_1\textrm{totwrk}+\beta_2\textrm{educ}+\beta_3\textrm{age}+u$

where sleep and total work are measured in minutes per week and educ and age are measured
in years.

### 1. If adults tradeoff sleep for work, what is the sign of $\beta_1$?

(Note: typing the words "beta 1" is fine. If you want to have it as math notation, type: \$ \beta_1 \$ . You can double click this cell and copy the code.)

If adults tradeoff sleep for work, we would expect a negative $\beta_1$, as the more work adults do, the less sleep they will have.

### 2. What signs to you think $\beta_2$ and $\beta_3$ will have?

Example Answer: I expect $\beta_2$ to be positive – holding constant age and total work, I expect increases in education to be associated with increases in sleep because people who are more educated will know more about the benefits of sleep and make sleep a priority. (You could also argue for the opposite sign).

I expect $\beta_3$ to be negative – holding constant education and total work, I expect increases in age to be associated with decreases in sleep because the human will likely find more things to distract herself/himself at night. (Again, you could also argue for the opposite sign).

Regardless of the sign you chose for each, remember that because you are dealing with multiple regression, you need to think of the sign of the beta you’re asked about _holding other variables in the model constant_!

### 3. Load in the dataset: sleep75.dta and estimate the above model. Display the results with the `summary()` command.  Remember that this is a .dta file, meaning that the `haven` package and `read_dta()` commands will be useful.

In [6]:
library(haven)
sleepdata<-read_dta("sleep75.dta")


sleepreg<-lm(sleep~totwrk+educ+age, data=sleepdata)
summary(sleepreg)


Call:
lm(formula = sleep ~ totwrk + educ + age, data = sleepdata)

Residuals:
     Min       1Q   Median       3Q      Max 
-2405.12  -236.50    14.53   253.87  1303.42 

Coefficients:
              Estimate Std. Error t value Pr(>|t|)    
(Intercept) 3638.24531  112.27511  32.405   <2e-16 ***
totwrk        -0.14837    0.01669  -8.888   <2e-16 ***
educ         -11.13381    5.88457  -1.892   0.0589 .  
age            2.19988    1.44572   1.522   0.1285    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 419.4 on 702 degrees of freedom
Multiple R-squared:  0.1134,	Adjusted R-squared:  0.1096 
F-statistic: 29.92 on 3 and 702 DF,  p-value: < 2.2e-16


### 4. Interpret the effect $\hat{\beta}_1 $.

An increase in 1 minute per week is associated with a decrease of 0.148 minutes per week of sleep, holding constant education and age.

### 5. Discuss whether assumptions MLR1-MLR4 are reasonable or may be violated in these data. 

MLR1– It seems plausible that the model linear in its parameters (no reason to think otherwise)

MLR2–Having a random sample is contingent on the sampling method, so it may be violated if the authors were very selective about how they chose their sample.

MLR3–None of the independent variables in the model are likely to be perfectlycollinear with one another. It is reasonable to expect that age, education, and total work do not perfectly explain one another. We can confirm this is true because we were able to estimate the model.

MLR4–It is likely the case that this assumption is not satisfied. We are perhaps not including something that matters within our model, such as number of people in thehousehold (where you might imagine that
the larger the household, the more noise people will make, and the less sleep one is likely able to have) that is also correlated with the included variables